src/components/Email.vue

```diff
@@ -38,6 +38,8 @@ export default {
         return 
       }
       await this.$store.commit('storeEmail', this.email);
```

Comment:
This invocation of `getSaved` might make more sense following the success message, rather than before, but we can come back to this if it ends up causing excessive load time.